src/hooks/useWalletCloudBackup.ts

```diff
@@ -34,6 +38,8 @@ function getUserError(e: Error) {
       return `We couldn't find your previous backup!`;
     case CLOUD_BACKUP_ERRORS.ERROR_GETTING_ENCRYPTED_DATA:
       return `We couldn't access your backup at this time. Please try again later.`;
+    case CLOUD_BACKUP_ERRORS.MISSING_PIN:
+      return `Something went wrong while processing your PIN code. Please try again later.`;
```

Comment:
If the pin is missing trying again later won't make a difference. I'd say contact our support team or something like that.

Reply:
But this is the PIN code that user enters in the PIN code screen, not the one that is store with the backup. For sure the message could be a bit more clear. You have any ides?

Reply:
Yeah I understand, I'm just pointing out that "try again later won't make a difference".